### PR TITLE
Use pkg-config to find fuse libraries

### PR DESCRIPTION
**Pull request by ardovm**
Should allow building on modern Linux hosts (partially fix of issue #4)

Please note that this is my own work, and is not based on the [Debian package patches](https://packages.debian.org/source/buster/rdiff-backup-fs).
(I read about the Debian package just after committing this)
Integrating the other patches from the Debian package is beyond my current purposes, and could be a possibly better alternative than merging this PR.

### CODE REVIEW

**Review by gruszczy — APPROVED** [review]

**Comment by ericzolf** [comment]
My suggestion:

* I merge once @anotherPuppet confirms that he can use it to compile the driver
* we keep the issue #4 for further improvements using the Debian patches

Opinions?

**Comment by ericzolf** [comment]
I meant (of course) @anotherMuppet (sorry!)

**Review by anotherMuppet — APPROVED** [review]
i can confirm compilation and mounting a repository results in correct file sizes for large files without i/o errors

**Comment by anotherMuppet** [comment]
i'm not sure how to reward you the $30 on bountysource, @ardovm

i think @ericzolf has to claim ownership of the project on bountysource and then myself or i can award you

**Comment by ardovm** [comment]
That's very kind of you... but do not worry. There is no urgency.

**Comment by anotherMuppet** [comment]
@ericzolf can you claim ownership of the rdiff-backup team on bountysource? https://www.bountysource.com/issues/96838952-files-with-a-byte-size-larger-than-an-int-cause-i-o-errors

i can't seem to navigate that particular site very well, and google isn't much help (nor is their documentation)

**Comment by anotherMuppet** [comment]
actually, @ardovm i think you need to visit the above link and click "solve issue" and then claim it, then i can reject or approve it

afaik there's nothing you need to do @ericzolf

**Comment by ardovm** [comment]
@anotherMuppet I did. I had to indicate a pull request and I indicated this one, because the previous PR #4 was already closed and, in fact, did not completely solve the problem (the code did not build).

So, maybe, merging this PR should be the next step in order to assign the bounty.

As I already wrote, I am not in a hurry. If there are other changes that you would like merged into the code, please tell me and I will update this PR. When all of you will be satisfied, we will merge it and proceed further.

**Comment by anotherMuppet** [comment]
@ardovm i've lodged a support ticket with bountysource, hopefully they'll get back to me soon

also of note, i think bountysource charges a 10% fee on withdrawals, so that's reduced it to $27

i'll leave it up to @ericzolf to do the merging, etc.

**Comment by anotherMuppet** [comment]
@ardovm support just got back to me and said it was due to a sync issue, you can now claim the bounty https://www.bountysource.com/issues/96838952-files-with-a-byte-size-larger-than-an-int-cause-i-o-errors

**Comment by ericzolf** [comment]
As a said that I'll merge once @anotherMuppet confirms, I merge. Thanks for this, it was fun!

**Pull request merged**
